### PR TITLE
[Binary-Package]The release binary package should contain seatunnel-starter

### PR DESCRIPTION
**Pull request by CalvinKirs**
And should contain a connector-v2 jars to source code publish
build result dir:
image

### CODE REVIEW

**Review by EricJoy2048 — DISMISSED** [review]
+1

File: pom.xml [hunk 1]
```diff
@@ -82,6 +82,7 @@
         <module>seatunnel-core</module>
         <module>seatunnel-transforms</module>
         <module>seatunnel-connectors</module>
+        <module>seatunnel-connectors-v2</module>
```
Comment (Hisoka-X) [inline pom.xml:85]: Remove it from profiles id with `all`

Comment (CalvinKirs) [reply pom.xml:85]: done

**Review by Hisoka-X — APPROVED** [review]

**Comment by hailin0** [comment]
LGTM

**Review by EricJoy2048 — COMMENTED** [review]
+1

**Review by EricJoy2048 — APPROVED** [review]
+1

**Pull request merged**